Basic-Car-Maintenance/Shared/Settings/Views/ContributorsProfileView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ContributorsProfileView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import SwiftUI
+
+struct ContributorsProfileViews: View {
+    
+    var name: String
+    var imgUrl: String
+    private let imgFrame: CGFloat = 30
+    
+    init(name: String, url: String) {
+        self.name = name
+        self.imgUrl = url
+        self.name = (name.first?.uppercased())! + name.dropFirst()
+    }
+    
+    var body: some View {
+        HStack {
+            AsyncImage(url: URL(string: imgUrl)!) { phase in
+                // This closure takes a `LoadableImage` phase and returns a View for each phase.
+                switch phase {
+                case .empty:
+                    // Placeholder view when the image is not loaded yet
+                    Circle()
+                        .frame(width: imgFrame, height: imgFrame)
+                        .redacted(reason: .placeholder)
+                        .foregroundColor(.gray)
+                case .success(let image):
+                    // View to display the loaded image
+                    image
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: imgFrame, height: imgFrame)
+                        .clipShape(Circle())
+                case .failure(_):
```

Comment:
Instead of `Text("Failed to load image")` show an SF symbol with `person` as the default. Same for the unknown state